ApollosProject/apollos-apps
# Add delete profile

## ✏️ Description

Add ability to delete profile per https://3.basecamp.com/3926363/buckets/27088350/card_tables/cards/5717402494. 

## 🔬 To Test
1. Verify the alert fires an API call to delete account
2. Verify that, once an account is deleted, the app "logs out"


https://user-images.githubusercontent.com/45675751/221470871-e3bbb62a-40e4-4b87-ac4b-14ea12482166.mp4
